lib/active_admin/dependency.rb

```diff
@@ -55,6 +55,10 @@ def self.[](name)
       Matcher.new name.to_s
     end
 
+    def self.rails5?
+      rails >= '5.beta'
```

Comment:
I'll need to look into what pattern we need here to also match against the RC releases.